root/documentation/vep.conf

```diff
@@ -32,7 +32,12 @@
         type=Boolean
         description=Include HGVS nomenclature based on Ensembl stable identifiers
         default=0
-      </hgvs> 
+      </hgvs>
+      <ambiguous_hgvs>
```

Comment:
This is not relevant for the vep/:species/region/:region/:allele/ endpoint as it does not take HGVS as input

Reply:
Removed documentation of `--ambiguous_hgvs` for endpoints that do not receive HGVS as input: https://github.com/Ensembl/ensembl-rest/pull/571/commits/c94340954284655e971b1012264e64354a2e885a